### ISSUE TITLE
[Feature]: Forcing a custom version when shipping

### ISSUE THREAD

**Issue by malmeloo** [pacstall/pacup.py#184]
### Is your feature related to a problem? Please describe.

Sometimes repology does not have the latest version yet or is simply incorrect. This is especially annoying for packages that simply refuse to start up until they are updated.

### A clear and concise description of what you want to happen.

If #18 gets implemented, that should already help fix the problem. However it would still be nice to be able to manually specify a version. I was thinking of the following syntax:

```
pacup -s some.pacscript@1.2.3 another.pacscript@4.5.6
```

### Describe alternatives you've considered

Manually updating the pacscript :(
or writing a script for it, but this feels like a feature that belongs in pacup.

**Comment by Zahrun** [comment]
+1

For the syntax it could also just be `pacup -s some.pacscript --to 4.5.6`

**Comment by malmeloo** [comment]
> For the syntax it could also just be `pacup -s some.pacscript --to 4.5.6`

I think the problem with that syntax is that it's technically possible to ship multiple packages at once, so using a flag would force all of them to the same version (unless you do some weird parsing trickery)

**Comment by Elsie19** [comment]
@D-Brox would you be able to handle this?

Maybe the syntax could be something like:
```bash
$ pacup --ship=4.5.6 some.pacscript
```

**Comment by Elsie19** [comment]
Will be done in the Perl rewrite, which will be released alongside 5.0.0.
